buddyboss/buddyboss-platform
# Discussion reply notification issue - fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2948  .

### How to test the changes in this Pull Request:

1. Let user1 create a discussion
2. Let user2 reply to the discussion
3. Notice that user1 will receive a notification
4. Let user1 reply to user2 in discussion
5. Notice that user2 will receive a notification now.

### Proof Screenshots or Video

https://www.loom.com/share/58c62cb39c794c60bba988735e422e76

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
